types/tableau-js-api/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for Tableau Javascript API 2.8
```

Comment:
Instead of turning off npm-naming in dtslint.json, please mark that this is a non-npm package in the header:

```suggestion
// Type definitions for non-npm package Tableau Javascript API 2.8
```